Installation/Action/synczaken.action.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title": "ZgwToVrijbrpBirthAction",
+  "$id": "https://vrijbrp.nl/action/vrijbrp.synczaken.action.json",
```

Comment:
different domain than mappings from https://github.com/CommonGateway/VrijBRPToZGWBundle/pull/1/files
might be better to be consistent with these references